src/Athena/Views/Shared/_Nav.cshtml

```diff
@@ -14,22 +14,20 @@
 }
 else
 {
-
             <li>
-                <a class="chip" href="@Url.Action(nameof(HomeController.User_Page), "Home")" tooltip="Student Page">
+                <a class="navChip chip" href="@Url.Action(nameof(HomeController.User_Page), "Home")" tooltip="Student Page">
```

Comment:
Do you really need a class for this? I think you can just use `nav .chip` as a selector in the stylesheet.